akka-stream/src/main/scala/akka/stream/scaladsl/Tcp.scala

```diff
@@ -276,8 +276,6 @@ final class Tcp(system: ExtendedActorSystem) extends akka.actor.Extension {
    * @see [[Tcp.outgoingConnection]]
    * @param negotiateNewSession Details about what to require when negotiating the connection with the server
    * @param sslContext Context containing details such as the trust and keystore
-   *
-   * Marked API-may-change to leave room for an improvement around the very long parameter list.
```

Comment:
Not sure about this removal (or the next one): the method is deprecated and doesn't have the `@ApiMayChange` annotation. This comment seems obsolete or no-longer necessary.